tracing/src/main/java/com/palantir/tracing/DeferredTracer.java

```diff
@@ -71,48 +60,28 @@ public DeferredTracer(String operation) {
      * If no operation is specified, will attempt to use the parent span's operation name.
      */
     public DeferredTracer(Optional<String> operation) {
-        Optional<Trace> maybeTrace = Tracer.copyTrace();
-        if (maybeTrace.isPresent()) {
-            Trace trace = maybeTrace.get();
-            this.traceId = trace.getTraceId();
-            this.isObservable = trace.isObservable();
-            this.parentSpanId = trace.top().map(OpenSpan::getSpanId).orElse(null);
-            this.operation = operation.orElse(trace.top().map(OpenSpan::getOperation).orElse(DEFAULT_OPERATION));
-        } else {
-            this.traceId = null;
-            this.isObservable = false;
-            this.parentSpanId = null;
-            this.operation = null;
-        }
+        this.operation = operation.orElse(DEFAULT_OPERATION);
```

Comment:
Forwarding @dfox's comment about the default operation name: https://github.com/palantir/tracing-java/pull/110#discussion_r269833764

Reply:
I missed this sorry!